src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -425,6 +425,32 @@ export class RoslynLanguageServer {
                 if (solutionUris) {
                     if (solutionUris.length === 1) {
                         this.openSolution(solutionUris[0]);
```

Comment:
Sorry I'm a bit late to the party. Since we are trying to improve this area, could we also show the solution picker if default solution can't be opened and there's other solution files in the hierachy?

> [Error - 1:58:27 PM] [LanguageServerHost]Microsoft.Build.Exceptions.InvalidProjectFileException: The project file could not be loaded. Could not find file 'c:\Users\gel\source\repos\ConsoleApp16\ConsoleApp16.sln'.  c:\Users\gel\source\repos\ConsoleApp16\ConsoleApp16.sln

Reply:
@genlu We _could_, although I'm still curious how you got an error like that in the first place. :smile: